recipes/unicycler/meta.yaml

```diff
@@ -1,40 +1,40 @@
-{% set name = "unicycler" %}
-{% set version = "0.4.2" %}
+{% set version = "0.4.6" %}
+{% set sha256 = "56f6f358a5d1f8dd0fcd1df04504079fc42cec8453a36ee59ff89295535d03f5" %}
 
 package:
-  name: {{ name|lower }}
-  version: {{ version }}
-
-build:
-  number: 1
-  skip: True
-  script: python -m pip install --no-deps --ignore-installed .
+  name: unicycler
+  version: '{{ version }}'
 
 source:
   url: https://github.com/rrwick/Unicycler/archive/v{{ version }}.tar.gz
-  sha256: 58dbeee3b829e00a6f53ea0a151fd6dc77f1a7c35fefa31602deb2f1f5f481ea
+  sha256: '{{ sha256 }}'
   patches:
     - Makefile.patch [linux]
     - misc.py.patch
 
+build:
+  number: 0
+  skip: true
```

Comment:
we could actually try to build this recipe now by overwriting the default compilers and using the new once